net-print/kyocera-1x2x-mfp-driver/kyocera-1x2x-mfp-driver-1.1203-r1.ebuild

```diff
@@ -0,0 +1,131 @@
+IUSE="+rastertokpsl-fix ${IUSE_LINGUAS[@]/#/linguas_}"
+REQUIRED_USE="|| ( ${IUSE_LINGUAS[@]/#/linguas_} )"
+
+RDEPEND="net-print/cups"
+DEPEND="app-arch/unzip"
+RESTRICT="fetch"
+
+S="${WORKDIR}/Linux"
+
+function archSubdirectory {
+	if use x86 && ! use amd64
+	then echo "32bit"
+	else echo "64bit" ; fi
+}
+
+function languageArchiveName {
+	# Note the capitalisation inconsistency. Don't "fix" that.
+	case "$1" in
+		ar)    echo arabic      ;;
+		cs)    echo czech       ;;
+		de)    echo German      ;;
+		el)    echo greek       ;;
+		en)    echo English     ;;
+		es)    echo Spanish     ;;
+		fr)    echo French      ;;
+		he)    echo hebrew      ;;
+		hu)    echo hungarian   ;;
+		it)    echo Italian     ;;
+		ko)    echo Korean      ;;
+		pl)    echo polish      ;;
+		pt)    echo Portuguese  ;;
+		ro)    echo romanian    ;;
+		ru)    echo russian     ;;
+		th)    echo thai        ;;
+		tr)    echo turkish     ;;
+		vi)    echo vietnamese  ;;
+		zh_CN) echo simplified  ;;
+		zh_TW) echo traditional ;;
+	esac
+}
+
+pkg_nofetch() {
+	einfo "Please, navigate your browser to the following URL and manually"
+	einfo "download the file named '$A', then put it into '$DISTDIR'."
+	einfo "http://www.kyoceradocumentsolutions.eu/index/service/dlc.false.driver.FS1025MFP._.EN.html"
+	einfo ""
+	einfo "Consider keeping a local copy of the file since there're chances"
+	einfo "the company is going to eventually stop hosting it for whatever"
+	einfo "reason."
+}
+
+src_prepare() {
+	eapply_user
+
+	function patchFilterPath {
+		local orig="/usr/lib/cups/filter/rastertokpsl"
+		local repl="/usr/libexec/cups/filter/rastertokpsl"
+		use rastertokpsl-fix &&
+			repl="${repl}-fix.sh"
+
+		sed -i "s|$orig|$repl|g" "$@"
+	}
+
+	local g="$S/$(archSubdirectory)/Global"
+	local lingua
+	for lingua in "${IUSE_LINGUAS[@]}"; do
+		use linguas_$lingua || continue
+
+		local language="$(languageArchiveName "$lingua")"
+
+		unpack "$g/$language.tar.gz"
+		patchFilterPath "$language"/Kyocera*.ppd
+
+		# The upstream contains a number of files named 'rastertokpsl',
+		# each having the same content.
+		mv --no-clobber "$language/rastertokpsl" "$S/rastertokpsl"
+	done
+
+	if use rastertokpsl-fix; then
+		cp "$FILESDIR"/rastertokpsl-fix.sh .
+	fi
+}
+
+src_install() {
+
+	function installWithLinguaSuffix {
+		local lingua="$1"; shift
+		for file in "$@"; do
+			local dn="$(dirname "$file")"
+			local bn="$(basename "$file" .ppd)"
+			newins "$dn/$bn.ppd" "${bn}_$lingua.ppd"
+		done
+	}
+
+	insinto /usr/share/cups/model/KyoceraMita
+	local lingua
+	for lingua in "${IUSE_LINGUAS[@]}"; do
+		use linguas_$lingua || continue
+
+		local language="$(languageArchiveName "$lingua")"
+		installWithLinguaSuffix "$lingua" \
+			"$language"/Kyocera*.ppd
+	done
+
+	exeinto /usr/libexec/cups/filter
+	doexe rastertokpsl
+	use rastertokpsl-fix &&
+		doexe rastertokpsl-fix.sh
+
+	dodoc "Readme.htm"
+
+	einfo "This packages installs Kyocera's 'Readme.htm' file (see package"
```

Comment:
I think you want `elog` here, since the user probably wants to see this after the install finished. `einfo` is meant for status messages.